pkg/kubectl/cmd/util/printing.go

```diff
@@ -30,6 +34,40 @@ func AddPrinterFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("template", "t", "", "Template string or path to template file to use when -o=template or -o=templatefile.  The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview]")
 }
 
+// AddOutputFlagsForMutation adds output related flags to a command. Used by mutations only.
+func AddOutputFlagsForMutation(cmd *cobra.Command) {
+	cmd.Flags().StringP("output", "o", "", "Output mode. Use \"-o name\" for shorter output (resource/name).")
```

Comment:
ValidateOutputArgs() in line 63 checks the -o flag args (can only be "" or "name"). Is this what you mean?